### ISSUE TITLE
Follow up on CDN CI upload rewrite

### ISSUE THREAD

**Issue by StachuDotNet** [darklang/dark#3994]
Follow-up on https://github.com/darklang/dark/issues/3984#issuecomment-1138631243

We're currently pushing all files successfully, but files beyond the root directory (e.g. `blazor/LibExecution.dll`) are being pushed to just `{BUCKET}/LibExecution-HASH.dll`, not `{BUCKET}/blazor/LibExeuction-HASH.dll`.

Also, we should add `-m` to ensure uploads are in parallel

**Comment by StachuDotNet** [comment]
> I hope it doesn't turn out to be painful to upload files to non-root directories in the BUCKET.

So far, this seems like I'm back in the place that I was upon realizing [this](https://github.com/darklang/dark/pull/3984#issuecomment-1137921354), which really stinks.

**Comment by pbiggar** [comment]
Maybe this:

```
  -R, -r         The ``-R`` and ``-r`` options are synonymous. They enable directories,
                 buckets, and bucket subdirectories to be copied recursively.
                 If you don't use this option for an upload, gsutil copies objects
                 it finds and skips directories. Similarly, if you don't
                 specify this option for a download, gsutil copies
                 objects at the current bucket directory level and skips subdirectories.
```

**Comment by StachuDotNet** [comment]
Maybe. Wondering if it can be paired with an explicit list of files to copy. Will test locally shortly.

Otherwise, another hack would be to:
- create a temp directory per content-type (e.g. make an `application/json` folder)
- push the relevant files to that directory
- call `gsutil cp` per each of those folders, with `-r`

Not clean, but better than broken.

**Comment by StachuDotNet** [comment]
How would -r work with explicit file lists?
If I pass in `gsutil cp -r {files_to_copy} "{BUCKET}"`, and `files_to_copy` contains files at `/tmp/static-assets/blazor`, not sure how it's supposed to figure out that `blazor` is the only part I care about.

Doesn't seem to work locally, anyway.

So, without a better idea, I'll go with the above hack for now.

**Comment by StachuDotNet** [comment]
Resolved in #3995
